packages/react-components/src/components/AppFrame/AppFrame.mdx

```diff
@@ -0,0 +1,164 @@
+
+#### `NavigationGroup` component
+
+The `NavigationGroup` component is a container responsible for the appropriate grouping of `NavigationItem` elements. Should be a direct child of the `Navigation` component. In addition to its container function, it enables the scrolling of elements when there is insufficient space for their display. This feature can be activated by passing the prop `scrollable={true}`.
+
+#### `NavigationItem` component
+
+The `NavigationItem` component is responsible for displaying navigation elements. It displays buttons with an icon (or any other element you provide, keeping in mind the size limitations of this component). Additionally, a tooltip displaying the name of the element can appear on hover, as specified by the `label` prop, this can be disabled if necessary. The badge prop is also optional.
+
+#### Navigation implementation example
+
+```jsx
+import { AppFrame, Navigation, NavigationGroup, NavigationItem, Icon } from '@livechat/design-system-react-components';
+import { Home, Settings } from '@livechat/design-system-icons';
+
+<AppFrame
+  navigation={
+    <Navigation>
+      // top navigation
+      <NavigationGroup scrollable>
+        ...
+        <NavigationItem
+          key="home"
+          id="home"
+          label="Home"
+          icon={<Icon source={Home} />}
+          onClick={(e, id) => {}}
+          isActive={activeItem === 'home'}
+          badge={5}
+          url="#"
+        />
+        ...
+      </NavigationGroup>
+      // bottom navigation
+      <NavigationGroup>
+        ...
+        <NavigationItem
+          key="settings"
+          id="settings"
+          label="Settings"
+          icon={<Icon source={Settings} />}
+          onClick={(e, id) => {}}
+          isActive={activeItem === 'settings'}
+          badge='alert'
+          url="#"
+        />
+        ...
+      </NavigationGroup>
+    </Navigation>
+  }
+>
+  App content
+</AppFrame>
+```
+
+### Side navigation <a id="SideNav" />
+
+To build the side navigation, use the available components:
+
+- `SideNavigation`
+- `SideNavigationGroup`
+- `SideNavigationItem`
+
+#### `SideNavigation` component
+
+It mainly serves as a container for side navigation elements. An addition in this component is props `title` which will display an additional header above the list of navigation buttons.
+
+#### `SideNavigationGroup` component
+
+Unlike `NavigationGroup` it is not a component required to create a side navigation structure. Its main task is to aggregate elements under a label, with the option of enabling expansion of the list of elements via the `isCollapsible` prop.
```

Comment:
In my opinion, it is more effective to first describe the purpose of this component to developers, rather than highlighting its differences from other components. Therefore, I suggest starting the description as follows: "The SideNavigationGroup component is used to create collapsible items within the SideNavigation. It groups the `SideNavigationItem` items under same category."